docs/src/ci-intro.md

```diff
@@ -489,6 +490,32 @@ The contents of the `$web` storage container can be accessed from a browser by u
 This step will not work for pull requests created from a forked repository because such workflow [doesn't have access to the secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow).
 :::
 
+## Publishing results on GitHub Actions Job Summaries
+
+You also have the option to view your test results directly in the GitHub Actions Job Summaries. This is useful for quickly checking the status of your tests without having to download the HTML report.
+
+To do this, you can make use of the [@estruyf/github-actions-reporter](https://www.npmjs.com/package/@estruyf/github-actions-reporter) reporter.
```

Comment:
We don't inline third party module recommendations as users might consider it to be coming from Playwright / Microsoft and assume the code is trusted. You can add a link to your reporter to docs/src/test-reporters-js.md's Third party reporter showcase

Reply:
I wasn't sure, thanks for clarifying. 

I'll check out the file you've mentioned.